packages/math-poolsimulator/src/gyroE/index.test.ts

```diff
@@ -10,76 +10,91 @@ import { ExtendedGyroEV2 } from "./index";
 
 describe("Tests new Gyro ECLP math function based on package other functions", () => {
   const poolData = poolsFromFile["pool"];
-  const percentages = [25, 50, 75, 95];
+  const percentages = [25, 50, 75, 80];
```

Comment:
@yvesfracari I don't know why, but the highest percentage to converge for GyroE was 80 🤔

Reply:
Hummm! That is strange, looking at it rn